opm/simulators/timestepping/TimeStepControl.cpp

```diff
@@ -141,17 +142,23 @@ namespace Opm
         errors_[ 2 ] = error;
         for( int i=0; i<2; ++i ) {
             assert(std::isfinite(errors_[i]));
-            assert(errors_[i]>0);
+            assert(errors_[i]>=0);
```

Comment:
This `assert()` is basically tautologous now.  The `RelativeChangeInterface` [stipulates](https://github.com/OPM/opm-simulators/blob/5daf355a7b25b52850404bf3fa96aedad14cf648/opm/simulators/timestepping/TimeStepControlInterface.hpp#L36-L37) that this relation always holds.  Whether we choose to keep the `assert()` is a philosophical discussion I think.

Reply:
You are right, it is not needed anymore.